test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -236,6 +236,19 @@ test_fail_indented_multi_line_comment:
     FROM
         global_actions_states
 
+test_jinja_with_disbalanced_pairs:
+  # The range(3) -%} results in swallowing the \n
+  # N.B. The way L003 handles this is questionable,
+  # and this test seals in that behaviour.
+  pass_str: |
+    SELECT
+        cohort_month
+        {% for i in range(3) -%}
+            , {{ i }} AS index_{{ i }}
+        {% endfor -%}
+        , TRUE AS overall
+    FROM orders
+
```

Comment:
OK I think I get what's questionable now. We effectively ignore the spacing.

I don't think this example is actually questionable, and potentially this comment is confusing and should be removed?

However, I agree this also means this following SQL is not flagging L003, when it probably should (and that is the questionable part I presume?):

```sql
SELECT
    cohort_month
    {% for i in range(3) -%}
        , {{ i }} AS index_{{ i }}
    {% endfor -%}
, TRUE AS overall
FROM orders
```

Given, that I wonder should we address that example? Happy to merge this in meantime but maybe should open a separate issue for that?

Reply:
Oh wait - what's questionable is the middle section! OK I get you now.